tests/MongoDB.Driver.Core.Tests/Core/Misc/ServerVersionTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using FluentAssertions;
+using Xunit;
+
+namespace MongoDB.Driver.Core.Misc
+{
+    public class ServerVersionTests
+    {
+        [Theory]
```

Comment:
I expected the SemanticVersionTests.cs and ServerVersionTests.cs to have exactly the same test cases (they almost do).

It's OK if ServerVersionTests.cs has a few additional test **methods** specific to ServerVersion.

Reply:
All the public methods from `ServerVersion` are covered.

Reply:
My point wasn't that they weren't covered. But that the test cases were not identical between SemanticVersion and ServerVersion.

I think they should be identical but it's not super important.